iotdb-1.1/src/main/java/cn/edu/tsinghua/iot/benchmark/iotdb110/IoTDBSessionBase.java

```diff
@@ -211,7 +212,13 @@ protected Status executeQueryAndGetStatus(String sql, Operation operation) {
                       }
                       records.add(record);
                     }
+                  } else {
+                    SessionDataSet.DataIterator iterator = sessionDataSet.iterator();
+                    while (iterator.next()) {
+                      line.getAndIncrement();
+                    }
                   }
+
```

Comment:
delete empty line ?

Reply:
empty line used to make the next close operation more readable.